server/utils/licenceStatus.js

```diff
@@ -195,12 +195,7 @@ function getCaStageState(licence) {
 }
 
 function getEligibilityStageState(licence) {
-  const { exclusion, excluded } = getExclusionState(licence)
-  const { crdTime, insufficientTime, insufficientTimeContinue, insufficientTimeStop } = getCrdTimeState(licence)
-  const { suitability, unsuitable, unsuitableResult, exceptionalCircumstances } = getSuitabilityState(licence)
-
-  const notEligible = excluded || insufficientTimeStop || unsuitableResult
-  const { eligibility, eligible } = getEligibilityState(notEligible, [exclusion, crdTime, suitability])
+  const eligibilityState = getEligibilityState(licence)
 
   const { curfewAddressApproved, addressReviewFailed } = getCurfewAddressReviewState(licence)
```

Comment:
it winds me up a bit that one of these is a task and one of these is a decision! its not at all obvious from reading it